public/test/examples/bundles/README.md

```diff
@@ -0,0 +1,12 @@
+This folder contains the sources for the bundles used in E2E tests.
```

Comment:
These additional README's are great. My only concern is that docs that aren't linked from the root README might be used less or get stale. What do you think about dropping a single line in the main README like `there are details on things like <source map bundling> and <building the CRA> in the test directory` under the `Testing` heading?

Reply:
I thought about it, but these are only necessary to rebuild the examples, which is extremely rare (this is the first time we had to do it and I don't expect it to happen again any time soon), so I think adding them to the root README would be a distraction.